src/wp-includes/class-wp-image-editor-gd.php

```diff
@@ -261,18 +261,20 @@ public function multi_resize( $sizes ) {
 	 * Create an image sub-size and return the image meta data value for it.
 	 *
 	 * @since 5.3.0
+	 * @since 6.0.0 Added the $mime_type parameter.
 	 *
-	 * @param array $size_data {
+	 * @param array  $size_data {
 	 *     Array of size data.
 	 *
 	 *     @type int  $width  The maximum width in pixels.
 	 *     @type int  $height The maximum height in pixels.
 	 *     @type bool $crop   Whether to crop the image to exact dimensions.
 	 * }
+	 * @param string $mime_type Optional. The mime type of the image.
 	 * @return array|WP_Error The image data array for inclusion in the `sizes` array in the image meta,
 	 *                        WP_Error object on error.
 	 */
-	public function make_subsize( $size_data ) {
+	public function make_subsize( $size_data, $mime_type = null ) {
```

Comment:
Just leaving another note here that this affects 3P plugins implementing custom image editors. We need to call this out in the dev note that any such plugins need to add support for the `$mime_type` parameter, otherwise users of those plugins will run into problems where instead of the additional MIME types the method would generate another version of the original MIME type.

Reply:
👍🏼  I will add a note to my trac ticket to call this out. I'll also review the repository to see what popular plugins might be affected.

Reply:
added to developer note section of draft trac ticket post